apps/src/lib/client/tx.rs

```diff
@@ -1218,28 +1217,17 @@ pub async fn broadcast_tx(
         _ => panic!("Cannot broadcast a dry-run transaction"),
     };
 
-    let websocket_timeout =
-        if let Ok(val) = env::var(ENV_VAR_ANOMA_TENDERMINT_WEBSOCKET_TIMEOUT) {
-            if let Ok(timeout) = val.parse::<u64>() {
-                Duration::new(timeout, 0)
-            } else {
-                Duration::new(300, 0)
-            }
-        } else {
-            Duration::new(300, 0)
-        };
-
-    let mut wrapper_tx_subscription = TendermintWebsocketClient::open(
-        WebSocketAddress::try_from(address.clone())?,
-        Some(websocket_timeout),
-    )?;
-
-    let response = wrapper_tx_subscription
-        .broadcast_tx_sync(tx.to_bytes().into())
-        .await
-        .map_err(|err| WsError::Response(format!("{:?}", err)))?;
+    tracing::debug!(
+        tendermint_rpc_address = ?address,
+        transaction = ?to_broadcast,
+        "Broadcasting transaction",
+    );
+    let rpc_cli = HttpClient::new(address)?;
 
-    wrapper_tx_subscription.close();
+    // TODO: configure an explicit timeout value? we need to hack away at
+    // `tendermint-rs` for this, which is currently using a hard-coded 30s
+    // timeout.
```

Comment:
yeah, let's maybe turn this in a low-prio issue - 30s should be ok for mempool inclusion for most cases